README.md

```diff
@@ -90,4 +90,10 @@ Clone this repository, then run the following commands:
 $> yarn build:cli
 ```
 
+Run your newly built CLI:
+
+```
+$> node ./packages/berry-cli/bin/berry.js
```

Comment:
Note that this shouldn't be needed. If you check the `.yarnrc` you'll see that we have a `yarn-path` settings that points towards `./scripts/run-yarn.js`, which in turns call either the local build if available, or the checked-in one if not. So just calling the global Yarn should work!

Reply:
Oh, I didn't know that ... nice! I went looking around for a `/bin` like the previous version and turned up this. I can amend this to only include a bit about how the `yarn-path` is set and let you handle the allowance in #173

Reply:
Sounds good 👍

Reply:
Updated :slightly_smiling_face: